exam/exam/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace exam
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите чётное натуральное число, которое болшьше или равно 4");
+            int n = int.Parse(Console.ReadLine());
+            if (NumberСheck(n))
+            {
+                for (int a = 2; a < n; a++)
+                {
+                    if (CheckSimpleNumber(a))
+                    {
+                        for (int b = 2; b < n; b++)
```

Comment:
Программа выполняет лишнюю работу и выводит лишнюю информацию. Ведь сложение перестановочно. Так что если 26 = 3 + 13, то вычислять и писать 26 = 13 + 3 уже излишне. Достаточно проверять второе слагаемое, не меньшее первого.
```suggestion
                        for (int b = a; b < n; b++)
```